prow/cmd/sinker/main.go

```diff
@@ -239,9 +275,13 @@ func (c *controller) clean() {
 			// Delete old finished or orphan pods. Don't quit if we fail to delete one.
 			if err := client.Delete(pod.ObjectMeta.Name, &metav1.DeleteOptions{}); err == nil {
 				c.logger.WithField("pod", pod.ObjectMeta.Name).Info("Deleted old completed pod.")
+				metrics.podsRemoved[reason]++
 			} else {
 				c.logger.WithField("pod", pod.ObjectMeta.Name).WithError(err).Error("Error deleting pod.")
 			}
 		}
 	}
+
+	metrics.finishedAt = time.Now()
+	c.logger.WithFields(metrics.logFields()).Info("Show sinkerReconciliationMetrics.")
```

Comment:
Ultra-nit: Make the message not mention type name. "Sinker reconciliation metrics", or even "Sinker reconciliation complete"?

Reply:
Done. Thx.